### PR TITLE
better upgrade procedure for new node system

### PR DESCRIPTION
**Pull request by alichaddad**
https://github.com/0-complexity/openvcloud/issues/1825

### CODE REVIEW

File: scripts/install/installer [hunk 1]
```diff
@@ -640,19 +640,25 @@ class JumpScale7(Node):
...
         self.remote.run("systemctl start {}".format(" ".join(self.get_services())))
 
-    def stop(self):
+    def stop(self, ays=False):
         self.log("Stop JumpScale services")
-        res = self.remote.run("which ays", check=False)
-        if res.exit_status:
+        if ays:
             self.remote.run("ays stop")
+            self.remote.run('rm -f /etc/systemd/system/jsagent_main.service')
```
Comment (grimpy) [inline scripts/install/installer]: before this you want to disable it aswel
systemctl disable jsagent_main ays
and remove ays aswell while you are at ti

File: scripts/install/installer [hunk 2]
```diff
@@ -632,27 +632,35 @@ class JumpScale7(Node):
         if "cpu" in self.roles or "storage" in self.roles:
             services.append("nginx")
         if "cpu" in self.roles:
-            services.append("libvirtlister")
+            services.append("libvirtlistener")
```
Comment (ashraffouda) [inline scripts/install/installer]: nginx also on stor node

Comment (alichaddad) [reply scripts/install/installer]: That's already the case.

Comment (ashraffouda) [reply scripts/install/installer]: yeah, sorry didn't notice

**Pull request merged**